realm/realm-library/src/main/cpp/objectserver_shared.hpp

```diff
@@ -101,8 +101,8 @@ class JniSession {
                     realm::SyncFileActionMetadata(manager,
                                                   realm::SyncFileActionMetadata::Action::HandleRealmForClientReset,
                                                   original_path,
-                                                  nullptr,
-                                                  nullptr,
+                                                  "",
```

Comment:
I don't quite understand how could this be useful if `url` and `user_identity` are empty string...